### DISCUSSION TITLE
生成交互式的视频

### DISCUSSION THREAD

**Discussion by jokerwjx** [WebAV-Tech/WebAV#150] [Q&A]
WebAV想要生成交互式的视频 有解决的方法吗

**Comment by hughfenghen** [comment]
参考 #60 ，请提供详细的描述信息，想解决的问题。

**Comment by hughfenghen** [comment]
@jokerwjx 你好，最近我也想探索一下交互式视频。
可以加微信（liujun_fenghen）或邮箱（hughfenghen@gmail.com）一起交流。
